gapic-generator-cloud/templates/cloud/client/method/def/_client.erb

```diff
@@ -4,13 +4,13 @@
 <%= indent method.doc_description, "# " %>
 #
 <%- end -%>
-# @param requests [Enumerable<<%= method.request_type %> | Hash>]
+# @param requests [Google::Gax::StreamInput, Enumerable<<%= method.request_type %> | Hash>]
 #   An enumerable of {<%= method.request_type %>} instances.
-# @param options [Google::Gax::CallOptions]
+# @param options [Google::Gax::ApiCall::Options, Hash]
 #   Overrides the default settings for this call, e.g, timeout, retries, etc.
 #
-# @yield [result, operation] Access the result along with the RPC operation
-# @yieldparam result [<%= method.return_type %>]
+# @yield [response, operation] Access the result along with the RPC operation
```

Comment:
nit: use response instead of result in the text for consistency? Should we note the concrete type of the `operation`?

Reply:
For consistency, yes. Now that we have a `request` argument, it seems more correct to have a `response` returned.

Reply:
> Should we note the concrete type of the `operation`?

The type is documented 2 lines below.